command/vault/helper/secret_deleter.go

```diff
@@ -0,0 +1,45 @@
+package helper
```

Comment:
this is a breaking, unrelated change to your PR title, please open a new PR for discussion on this